internal/flushcommon/pipeline/flow_graph_embedding_node.go

```diff
@@ -168,11 +154,11 @@ func (eNode *embeddingNode) Operate(in []Msg) []Msg {
 	return []Msg{fgMsg}
 }
 
-func BuildSparseFieldData(dim int64, data [][]byte) storage.FieldData {
+func BuildSparseFieldData(array *schemapb.SparseFloatArray) storage.FieldData {
 	return &storage.SparseFloatVectorFieldData{
 		SparseFloatArray: schemapb.SparseFloatArray{
-			Contents: data,
-			Dim:      dim,
+			Contents: array.GetContents(),
```

Comment:
since the only member of `SparseFloatVectorFieldData` is same as the argument `schemapb.SparseFloatArray`, can we do this?

```
return &storage.SparseFloatVectorFieldData{
    SparseFloatArray: *array
}
```

maybe we even don't need such a function